src/Pages/AboutPage.jsx

```diff
@@ -3,92 +3,297 @@ import {
-          <Typography style={{ marginBottom: "1rem" }} variant="h2">
-            Our Team
-          </Typography>
-          <Typography variant="h5">
-            We are a group of dedicated volunteers working together to create
-            tools that help people navigate the conviction vacation process. We
-            are a diverse crew, drawn together by a drive for social justice.
-          </Typography>
-        </Box>
-        <Grid container justifyContent="center" spacing={3}>
-          {team.map((id) => (
-            <Grid item key={id}>
-              <TeamCard teamNumber={id} />
+      <Box className={classes.roots}>
+        <Container maxWidth="lg" className={classes.rootsContainer}>
+          <Typography variant="h2">The roots of our project</Typography>
+
+          <List>
+            <ListItemMobileSnap
+              image={
+                <Box
+                  component="img"
+                  src={seattle}
+                  className={classes.rootsImage}
+                />
+              }
+              text={
+                <Typography className={classes.rootsBodyText} variant="body1">
+                  One in four Washingtonians have been involved in the criminal
+                  justice system. Those with a criminal record face significant
+                  barriers to daily life after completing their prison terms.
+                </Typography>
+              }
+            />
+            <ListItemMobileSnap
+              image={
+                <Box
+                  component="img"
+                  src={lawyer}
+                  className={classes.rootsImage}
+                />
+              }
+              text={
+                <Typography className={classes.rootsBodyText} variant="body1">
+                  Washington State’s New Hope Act makes it easier for people
+                  with past criminal records to have their convictions vacated.
+                  Yet the system is slow and inefficient.
+                </Typography>
+              }
+              textLeft
+            />
+
+            <ListItemMobileSnap
+              image={
+                <Box
+                  component="img"
+                  src={innovation}
+                  className={classes.rootsImage}
+                />
+              }
+              text={
+                <Typography className={classes.rootsBodyText} variant="body1">
+                  We have identified opportunities to use technology to
+                  streamline this process and want to use our resources to
+                  appropriately meet user needs, creating an easier way to vacate
+                  convictions.
+                </Typography>
+              }
+            />
+          </List>
+        </Container>
+      </Box>
+      <Box className={classes.problemRoot}>
+        <Container maxWidth="lg">
+          <Grid container>
+            <Grid className={classes.theProblemRootleft} item sm={12} md={6}>
```

Comment:
Could we decrease the top padding _slightly_ here? Here's an image for reference

![Uploading Screen Shot 2022-01-17 at 9.57.23 AM.png…]()

Reply:
Not sure the reference uploaded properly.

Reply:
Let me know what you think of the newer padding.

Reply:
Looks great! 😄